pkgs/top-level/all-packages.nix

```diff
@@ -12577,7 +12577,6 @@ with pkgs;
   linux_testing = callPackage ../os-specific/linux/kernel/linux-testing.nix {
     kernelPatches = [
       kernelPatches.bridge_stp_helper
-      kernelPatches.p9_fixes
```

Comment:
Are those upstream now?

Reply:
As I understand, they are with this commit. https://github.com/torvalds/linux/commit/d18bee424b129aa4755268feeeb1ee16cbde6afa

Reply:
Yeah that's right, after two months of pings I got them in.